### PR TITLE
Add current and expected jsons with transformer code for edx.user.settings.changed

### PR DESCRIPTION
**Pull request by HamzaFarooqArbisoft**
**Trello Link:** _https://trello.com/c/UqodlTT9/127-user-settings-edxusersettingschanged_

**Description:** _Event occurs when you change something in your account page_

**Checks before merge:**

- [ ] Reviewed
- [ ] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/tests/expected/edx.user.settings.changed.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        "extensions": {
+            "new": 2005,
+            "old": null,
+            "setting": "year_of_birth",
+            "table": "auth_userprofile",
+            "truncated": [],
+            "user_id": 2
+        },
+        "id": "http://localhost:18000/u/honor",
+        "type": "Person"
```
Comment (asamolion) [inline openedx/features/caliper_tracking/tests/expected/edx.user.settings.changed.json]: Is there no type more appropriaate?

File: openedx/features/caliper_tracking/transformers/user_settings_transformers.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+"""
+Transformers for all the user setting events
+"""
+
+
+def edx_user_settings_changed(current_event, caliper_event):
+    """
+    Event occur when user change something in its Account Settings Page.
```
Comment (asamolion) [inline openedx/features/caliper_tracking/transformers/user_settings_transformers.py]: `Event occurs when a user changes in his/her account settings page.`

File: openedx/features/caliper_tracking/tests/expected/edx.user.settings.changed.json (hunk 1 above)
Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/tests/expected/edx.user.settings.changed.json]: I think, person should be the object as there is no other appropriate type for this event.

File: openedx/features/caliper_tracking/transformers/user_settings_transformers.py (hunk 2 above)
Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/transformers/user_settings_transformers.py]: done

File: openedx/features/caliper_tracking/tests/expected/edx.user.settings.changed.json (hunk 1 above)
Comment (asamolion) [reply openedx/features/caliper_tracking/tests/expected/edx.user.settings.changed.json]: Alright.

File: openedx/features/caliper_tracking/transformers/user_settings_transformers.py (hunk 2 above)
Comment (asamolion) [reply openedx/features/caliper_tracking/transformers/user_settings_transformers.py]: `Event occurs when a user changes anything in his/her account settings page.`

Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/transformers/user_settings_transformers.py]: done

**Pull request merged**